pkg/adapter/runtime.go

```diff
@@ -27,7 +27,7 @@ import (
 	"github.com/containers/libpod/pkg/util"
 	"github.com/containers/storage/pkg/archive"
 	"github.com/pkg/errors"
-	"k8s.io/api/core/v1"
+	v1 "k8s.io/api/core/v1"
```

Comment:
Why is this change required?

Reply:
The format-on-save feature of my editor automatically added it but it is not required or necessary. I removed it.